styles/dark-serif.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schemas.wp.org/wp/6.2/theme.json",
+    "version": 2,
+    "title": "Dark serif",
+    "settings": {
+      "color": {
+        "palette": [
+            {
+                "color": "#222222",
+                "name": "Base",
+                "slug": "base"
+            },
+            {
+                "color": "#ffffff",
+                "name": "Contrast",
+                "slug": "contrast"
+            },
+            {
+                "color": "#222222",
```

Comment:
I don't think we should be duplicating colors here, it's confusing for the user. Will we break patterns or templates if we remove it?

Reply:
Will not break but will get contrast color from default style

Reply:
Yea, we should not be duplicated color values. It's not helpful having two controls in the color palette UI for selecting the same color.